### DISCUSSION TITLE
umi 开启 mako 配置

### DISCUSSION THREAD

**Discussion by xijunking** [umijs/umi#12534] [Q&A]
我根据这个步骤尝试开启mako配置，但是出现了报错，想问一下有没有遇到过这个坑的，或者有没有什么排查思路，感谢🙇‍
![image](https://github.com/umijs/umi/assets/119270390/cf4ed992-a4f3-4ed6-b0d3-ad3390614adf)

![image](https://github.com/umijs/umi/assets/119270390/f7a08459-7217-485f-bd42-f7a0cf9be343)

**Comment by fz6m** [comment]
运行这个命令没遇到你的问题，试试 `pnpm umi config set mako "{}"` 呢。

如还有问题，需要给一个最小复现。

**Reply by fz6m** [reply]
你是 windows 吗，See https://github.com/umijs/mako/issues/1343

**Reply by xijunking** [reply]
感谢，我确实是windows
